browser/templates/browser/investigator.html

```diff
@@ -79,6 +79,26 @@
         </div>
     </div>
     {% endwith %}
+
+    {% with form.institution as field %}
+    <div class="row">
+        <div class="col-xs-12">
+            {% with form.institution as field %}
+                <div class="form-group">
+                    {% for error in field.errors %}
+                    <div class="alert alert-warning">{{ error }}</div>
+                    {% endfor %}
+                    <label>{{ field.label }}</label>
+                    {{ form.institution_search|addcss:"form-control input-sm" }}
+                    <div class="list-group" id="institution-results-container"></div>
+                    {{ field|addcss:"form-control input-sm" }}
```

Comment:
what is this input field used for?

Reply:
The input field is not required. Hence, removed this from the html file.